src/components/collection/Summary.js

```diff
@@ -77,12 +86,27 @@ function Summary({
   const categoriesDisplayed = allCategories?.filter((category) =>
     categories?.map((entry) => entry.categoryId).includes(category.id),
   );
+  const { data: member, isLoading: isMemberLoading } = hooks.useCurrentMember();
+  const { data: likedItems } = hooks.useLikedItems(member?.get('id'));
+
   const { mutate: postFlagItem } = useMutation(MUTATION_KEYS.POST_ITEM_FLAG);
+  const { mutate: updateFavoriteItem } = useMutation(MUTATION_KEYS.EDIT_MEMBER);
+  const { mutate: postItemLike } = useMutation(MUTATION_KEYS.POST_ITEM_LIKE);
+  const { mutate: deleteItemLike } = useMutation(
+    MUTATION_KEYS.DELETE_ITEM_LIKE,
+  );
+
   const [open, setOpen] = useState(false);
   const [selectedFlag, setSelectedFlag] = useState(false);
 
   const { data: flags } = hooks.useFlags();
 
+  if (isMemberLoading) return null;
```

Comment:
Show loader? I don't think it's a good to display nothing to the user on load.

Reply:
Or you don't need to check this if you check if the variable exists (as you did below).